spec/other/cli_dispatch_spec.rb

```diff
@@ -14,7 +14,7 @@
   end
 
   it "print a friendly error when ambiguous" do
-    bundle "i"
+    bundle "in"
```

Comment:
I'm not sure why it's on this branch, though. I committed it directly to `1-10-stable`. @andremedeiros looks like you cherry-picked the commit?